src/entry.asm

```diff
@@ -1,5 +1,11 @@
 include "sdk/hardware.inc"
 
+SECTION "gbsdk_systeminfo_ram", WRAM0
```

Comment:
In theory, if only one of the `CGB` and `SGB` and `DMG` defines is set, you won't need these and can save two bytes of RAM.

Reply:
Well, even if only one of those were set, theoretically someone could want to distinguish between DMG and MGB, or detect a CGB running in DMG mode or something like that. I could certainly add an `IF CGB` around `system_is_gba` though to save a byte.